west.yml

```diff
@@ -52,7 +52,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: a4edab554f67590a89703067af703c7996604562
+      revision: pull/738/head
```

Comment:
Please update SHA: `22e3f8090e8439b71b70c4b308cec98bb201776c` (and rebase).

Reply:
Done